app/views/docs/_methods.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2 id="methods">Methods</h2>
+
+<% @swagger.paths.each do |path| %>
+<% path.operations.each do |v| %>
```

Comment:
can we use `value` instead of `v` -- single character vars are hard to understand sometimes